(node:30604) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
(node:30604) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Error al buscar usuario: QueryFailedError: Error: Invalid column name 'Consignacion'.
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\typeorm\driver\sqlserver\SqlServerQueryRunner.js:200:30
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\base\request.js:440:25
    at Request.userCallback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:492:15)
    at Request.callback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\request.js:239:14)
    at Parser.onEndOfMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:2653:22)
    at Object.onceWrapper (node:events:632:28)
    at Parser.emit (node:events:518:28)
    at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:25:12)
    at Readable.emit (node:events:518:28)
    at endReadableNT (node:internal/streams/readable:1698:12) {
  query: 'SELECT "Usuario"."idUsuario" AS "Usuario_idUsuario", "Usuario"."Nombre" AS "Usuario_Nombre", "Usuario"."Clave" AS "Usuario_Clave", "Usuario"."idGrupo" AS "Usuario_idGrupo", "Usuario"."Activo" AS "Usuario_Activo", "Usuario"."FechaCierre" AS "Usuario_FechaCierre", "Usuario"."Credito" AS "Usuario_Credito", "Usuario"."Consignacion" AS "Usuario_Consignacion" FROM "Usuario" "Usuario" WHERE ("Usuario"."Nombre" = @0 AND "Usuario"."Activo" = @1) ORDER BY (SELECT NULL) OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY',
  parameters: [ 'A3CAJA', 1 ],
  driverError: RequestError: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    code: 'EREQUEST',
    originalError: Error: Invalid column name 'Consignacion'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      info: [ErrorMessageToken]
    },
    number: 207,
    lineNumber: 1,
    state: 1,
    class: 16,
    serverName: 'BDNODO01\\POINTDB',
    procName: '',
    precedingErrors: [ [RequestError], [RequestError] ]
  },
  code: 'EREQUEST',
  originalError: Error: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 207,
      state: 1,
      class: 16,
      message: "Invalid column name 'Consignacion'.",
      serverName: 'BDNODO01\\POINTDB',
      procName: '',
      lineNumber: 1
    }
  },
  number: 207,
  lineNumber: 1,
  state: 1,
  class: 16,
  serverName: 'BDNODO01\\POINTDB',
  procName: '',
  precedingErrors: [
    RequestError: Invalid column name 'FechaCierre'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    },
    RequestError: Invalid column name 'Credito'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    }
  ]
}
Error al buscar usuario: QueryFailedError: Error: Invalid column name 'Consignacion'.
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\typeorm\driver\sqlserver\SqlServerQueryRunner.js:200:30
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\base\request.js:440:25
    at Request.userCallback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:492:15)
    at Request.callback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\request.js:239:14)
    at Parser.onEndOfMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:2653:22)
    at Object.onceWrapper (node:events:632:28)
    at Parser.emit (node:events:518:28)
    at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:25:12)
    at Readable.emit (node:events:518:28)
    at endReadableNT (node:internal/streams/readable:1698:12) {
  query: 'SELECT "Usuario"."idUsuario" AS "Usuario_idUsuario", "Usuario"."Nombre" AS "Usuario_Nombre", "Usuario"."Clave" AS "Usuario_Clave", "Usuario"."idGrupo" AS "Usuario_idGrupo", "Usuario"."Activo" AS "Usuario_Activo", "Usuario"."FechaCierre" AS "Usuario_FechaCierre", "Usuario"."Credito" AS "Usuario_Credito", "Usuario"."Consignacion" AS "Usuario_Consignacion" FROM "Usuario" "Usuario" WHERE ("Usuario"."Nombre" = @0 AND "Usuario"."Activo" = @1) ORDER BY (SELECT NULL) OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY',
  parameters: [ 'ECEPEDA', 1 ],
  driverError: RequestError: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    code: 'EREQUEST',
    originalError: Error: Invalid column name 'Consignacion'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      info: [ErrorMessageToken]
    },
    number: 207,
    lineNumber: 1,
    state: 1,
    class: 16,
    serverName: 'BDNODO01\\POINTDB',
    procName: '',
    precedingErrors: [ [RequestError], [RequestError] ]
  },
  code: 'EREQUEST',
  originalError: Error: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 207,
      state: 1,
      class: 16,
      message: "Invalid column name 'Consignacion'.",
      serverName: 'BDNODO01\\POINTDB',
      procName: '',
      lineNumber: 1
    }
  },
  number: 207,
  lineNumber: 1,
  state: 1,
  class: 16,
  serverName: 'BDNODO01\\POINTDB',
  procName: '',
  precedingErrors: [
    RequestError: Invalid column name 'FechaCierre'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    },
    RequestError: Invalid column name 'Credito'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    }
  ]
}
Error al buscar usuario: QueryFailedError: Error: Invalid column name 'Consignacion'.
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\typeorm\driver\sqlserver\SqlServerQueryRunner.js:200:30
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\base\request.js:440:25
    at Request.userCallback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:492:15)
    at Request.callback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\request.js:239:14)
    at Parser.onEndOfMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:2653:22)
    at Object.onceWrapper (node:events:632:28)
    at Parser.emit (node:events:518:28)
    at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:25:12)
    at Readable.emit (node:events:518:28)
    at endReadableNT (node:internal/streams/readable:1698:12) {
  query: 'SELECT "Usuario"."idUsuario" AS "Usuario_idUsuario", "Usuario"."Nombre" AS "Usuario_Nombre", "Usuario"."Clave" AS "Usuario_Clave", "Usuario"."idGrupo" AS "Usuario_idGrupo", "Usuario"."Activo" AS "Usuario_Activo", "Usuario"."FechaCierre" AS "Usuario_FechaCierre", "Usuario"."Credito" AS "Usuario_Credito", "Usuario"."Consignacion" AS "Usuario_Consignacion" FROM "Usuario" "Usuario" WHERE ("Usuario"."Nombre" = @0 AND "Usuario"."Activo" = @1) ORDER BY (SELECT NULL) OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY',
  parameters: [ 'ECEPEDA', 1 ],
  driverError: RequestError: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    code: 'EREQUEST',
    originalError: Error: Invalid column name 'Consignacion'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      info: [ErrorMessageToken]
    },
    number: 207,
    lineNumber: 1,
    state: 1,
    class: 16,
    serverName: 'BDNODO01\\POINTDB',
    procName: '',
    precedingErrors: [ [RequestError], [RequestError] ]
  },
  code: 'EREQUEST',
  originalError: Error: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 207,
      state: 1,
      class: 16,
      message: "Invalid column name 'Consignacion'.",
      serverName: 'BDNODO01\\POINTDB',
      procName: '',
      lineNumber: 1
    }
  },
  number: 207,
  lineNumber: 1,
  state: 1,
  class: 16,
  serverName: 'BDNODO01\\POINTDB',
  procName: '',
  precedingErrors: [
    RequestError: Invalid column name 'FechaCierre'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    },
    RequestError: Invalid column name 'Credito'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    }
  ]
}
Error al buscar usuario: QueryFailedError: Error: Invalid column name 'Consignacion'.
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\typeorm\driver\sqlserver\SqlServerQueryRunner.js:200:30
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\base\request.js:440:25
    at Request.userCallback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:492:15)
    at Request.callback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\request.js:239:14)
    at Parser.onEndOfMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:2653:22)
    at Object.onceWrapper (node:events:632:28)
    at Parser.emit (node:events:518:28)
    at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:25:12)
    at Readable.emit (node:events:518:28)
    at endReadableNT (node:internal/streams/readable:1698:12) {
  query: 'SELECT "Usuario"."idUsuario" AS "Usuario_idUsuario", "Usuario"."Nombre" AS "Usuario_Nombre", "Usuario"."Clave" AS "Usuario_Clave", "Usuario"."idGrupo" AS "Usuario_idGrupo", "Usuario"."Activo" AS "Usuario_Activo", "Usuario"."FechaCierre" AS "Usuario_FechaCierre", "Usuario"."Credito" AS "Usuario_Credito", "Usuario"."Consignacion" AS "Usuario_Consignacion" FROM "Usuario" "Usuario" WHERE ("Usuario"."Nombre" = @0 AND "Usuario"."Activo" = @1) ORDER BY (SELECT NULL) OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY',
  parameters: [ 'A3CAJA', 1 ],
  driverError: RequestError: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    code: 'EREQUEST',
    originalError: Error: Invalid column name 'Consignacion'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      info: [ErrorMessageToken]
    },
    number: 207,
    lineNumber: 1,
    state: 1,
    class: 16,
    serverName: 'BDNODO01\\POINTDB',
    procName: '',
    precedingErrors: [ [RequestError], [RequestError] ]
  },
  code: 'EREQUEST',
  originalError: Error: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 207,
      state: 1,
      class: 16,
      message: "Invalid column name 'Consignacion'.",
      serverName: 'BDNODO01\\POINTDB',
      procName: '',
      lineNumber: 1
    }
  },
  number: 207,
  lineNumber: 1,
  state: 1,
  class: 16,
  serverName: 'BDNODO01\\POINTDB',
  procName: '',
  precedingErrors: [
    RequestError: Invalid column name 'FechaCierre'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    },
    RequestError: Invalid column name 'Credito'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    }
  ]
}
Error al buscar usuario: QueryFailedError: Error: Invalid column name 'Consignacion'.
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\typeorm\driver\sqlserver\SqlServerQueryRunner.js:200:30
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\base\request.js:440:25
    at Request.userCallback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:492:15)
    at Request.callback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\request.js:239:14)
    at Parser.onEndOfMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:2653:22)
    at Object.onceWrapper (node:events:632:28)
    at Parser.emit (node:events:518:28)
    at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:25:12)
    at Readable.emit (node:events:518:28)
    at endReadableNT (node:internal/streams/readable:1698:12) {
  query: 'SELECT "Usuario"."idUsuario" AS "Usuario_idUsuario", "Usuario"."Nombre" AS "Usuario_Nombre", "Usuario"."Clave" AS "Usuario_Clave", "Usuario"."idGrupo" AS "Usuario_idGrupo", "Usuario"."Activo" AS "Usuario_Activo", "Usuario"."FechaCierre" AS "Usuario_FechaCierre", "Usuario"."Credito" AS "Usuario_Credito", "Usuario"."Consignacion" AS "Usuario_Consignacion" FROM "Usuario" "Usuario" WHERE ("Usuario"."Nombre" = @0 AND "Usuario"."Activo" = @1) ORDER BY (SELECT NULL) OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY',
  parameters: [ 'A3CAJA', 1 ],
  driverError: RequestError: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    code: 'EREQUEST',
    originalError: Error: Invalid column name 'Consignacion'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      info: [ErrorMessageToken]
    },
    number: 207,
    lineNumber: 1,
    state: 1,
    class: 16,
    serverName: 'BDNODO01\\POINTDB',
    procName: '',
    precedingErrors: [ [RequestError], [RequestError] ]
  },
  code: 'EREQUEST',
  originalError: Error: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 207,
      state: 1,
      class: 16,
      message: "Invalid column name 'Consignacion'.",
      serverName: 'BDNODO01\\POINTDB',
      procName: '',
      lineNumber: 1
    }
  },
  number: 207,
  lineNumber: 1,
  state: 1,
  class: 16,
  serverName: 'BDNODO01\\POINTDB',
  procName: '',
  precedingErrors: [
    RequestError: Invalid column name 'FechaCierre'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    },
    RequestError: Invalid column name 'Credito'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    }
  ]
}
Error al buscar usuario: QueryFailedError: Error: Invalid column name 'Consignacion'.
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\typeorm\driver\sqlserver\SqlServerQueryRunner.js:200:30
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\base\request.js:440:25
    at Request.userCallback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:492:15)
    at Request.callback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\request.js:239:14)
    at Parser.onEndOfMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:2653:22)
    at Object.onceWrapper (node:events:632:28)
    at Parser.emit (node:events:518:28)
    at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:25:12)
    at Readable.emit (node:events:518:28)
    at endReadableNT (node:internal/streams/readable:1698:12) {
  query: 'SELECT "Usuario"."idUsuario" AS "Usuario_idUsuario", "Usuario"."Nombre" AS "Usuario_Nombre", "Usuario"."Clave" AS "Usuario_Clave", "Usuario"."idGrupo" AS "Usuario_idGrupo", "Usuario"."Activo" AS "Usuario_Activo", "Usuario"."FechaCierre" AS "Usuario_FechaCierre", "Usuario"."Credito" AS "Usuario_Credito", "Usuario"."Consignacion" AS "Usuario_Consignacion" FROM "Usuario" "Usuario" WHERE ("Usuario"."Nombre" = @0 AND "Usuario"."Activo" = @1) ORDER BY (SELECT NULL) OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY',
  parameters: [ 'A3CAJA', 1 ],
  driverError: RequestError: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    code: 'EREQUEST',
    originalError: Error: Invalid column name 'Consignacion'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      info: [ErrorMessageToken]
    },
    number: 207,
    lineNumber: 1,
    state: 1,
    class: 16,
    serverName: 'BDNODO01\\POINTDB',
    procName: '',
    precedingErrors: [ [RequestError], [RequestError] ]
  },
  code: 'EREQUEST',
  originalError: Error: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 207,
      state: 1,
      class: 16,
      message: "Invalid column name 'Consignacion'.",
      serverName: 'BDNODO01\\POINTDB',
      procName: '',
      lineNumber: 1
    }
  },
  number: 207,
  lineNumber: 1,
  state: 1,
  class: 16,
  serverName: 'BDNODO01\\POINTDB',
  procName: '',
  precedingErrors: [
    RequestError: Invalid column name 'FechaCierre'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    },
    RequestError: Invalid column name 'Credito'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    }
  ]
}
Error al buscar usuario: QueryFailedError: Error: Invalid column name 'Consignacion'.
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\typeorm\driver\sqlserver\SqlServerQueryRunner.js:200:30
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\base\request.js:440:25
    at Request.userCallback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:492:15)
    at Request.callback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\request.js:239:14)
    at Parser.onEndOfMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:2653:22)
    at Object.onceWrapper (node:events:632:28)
    at Parser.emit (node:events:518:28)
    at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:25:12)
    at Readable.emit (node:events:518:28)
    at endReadableNT (node:internal/streams/readable:1698:12) {
  query: 'SELECT "Usuario"."idUsuario" AS "Usuario_idUsuario", "Usuario"."Nombre" AS "Usuario_Nombre", "Usuario"."Clave" AS "Usuario_Clave", "Usuario"."idGrupo" AS "Usuario_idGrupo", "Usuario"."Activo" AS "Usuario_Activo", "Usuario"."FechaCierre" AS "Usuario_FechaCierre", "Usuario"."Credito" AS "Usuario_Credito", "Usuario"."Consignacion" AS "Usuario_Consignacion" FROM "Usuario" "Usuario" WHERE ("Usuario"."Nombre" = @0 AND "Usuario"."Activo" = @1) ORDER BY (SELECT NULL) OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY',
  parameters: [ 'A3CAJA', 1 ],
  driverError: RequestError: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    code: 'EREQUEST',
    originalError: Error: Invalid column name 'Consignacion'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      info: [ErrorMessageToken]
    },
    number: 207,
    lineNumber: 1,
    state: 1,
    class: 16,
    serverName: 'BDNODO01\\POINTDB',
    procName: '',
    precedingErrors: [ [RequestError], [RequestError] ]
  },
  code: 'EREQUEST',
  originalError: Error: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 207,
      state: 1,
      class: 16,
      message: "Invalid column name 'Consignacion'.",
      serverName: 'BDNODO01\\POINTDB',
      procName: '',
      lineNumber: 1
    }
  },
  number: 207,
  lineNumber: 1,
  state: 1,
  class: 16,
  serverName: 'BDNODO01\\POINTDB',
  procName: '',
  precedingErrors: [
    RequestError: Invalid column name 'FechaCierre'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    },
    RequestError: Invalid column name 'Credito'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    }
  ]
}
Error al buscar usuario: QueryFailedError: Error: Invalid column name 'Consignacion'.
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\typeorm\driver\sqlserver\SqlServerQueryRunner.js:200:30
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\base\request.js:440:25
    at Request.userCallback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:492:15)
    at Request.callback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\request.js:239:14)
    at Parser.onEndOfMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:2653:22)
    at Object.onceWrapper (node:events:632:28)
    at Parser.emit (node:events:518:28)
    at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:25:12)
    at Readable.emit (node:events:518:28)
    at endReadableNT (node:internal/streams/readable:1698:12) {
  query: 'SELECT "Usuario"."idUsuario" AS "Usuario_idUsuario", "Usuario"."Nombre" AS "Usuario_Nombre", "Usuario"."Clave" AS "Usuario_Clave", "Usuario"."idGrupo" AS "Usuario_idGrupo", "Usuario"."Activo" AS "Usuario_Activo", "Usuario"."FechaCierre" AS "Usuario_FechaCierre", "Usuario"."Credito" AS "Usuario_Credito", "Usuario"."Consignacion" AS "Usuario_Consignacion" FROM "Usuario" "Usuario" WHERE ("Usuario"."Nombre" = @0 AND "Usuario"."Activo" = @1 AND "Usuario"."Clave" = @2) ORDER BY (SELECT NULL) OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY',
  parameters: [ 'KHERAS', 1, '1600567265' ],
  driverError: RequestError: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    code: 'EREQUEST',
    originalError: Error: Invalid column name 'Consignacion'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      info: [ErrorMessageToken]
    },
    number: 207,
    lineNumber: 1,
    state: 1,
    class: 16,
    serverName: 'BDNODO01\\POINTDB',
    procName: '',
    precedingErrors: [ [RequestError], [RequestError] ]
  },
  code: 'EREQUEST',
  originalError: Error: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 207,
      state: 1,
      class: 16,
      message: "Invalid column name 'Consignacion'.",
      serverName: 'BDNODO01\\POINTDB',
      procName: '',
      lineNumber: 1
    }
  },
  number: 207,
  lineNumber: 1,
  state: 1,
  class: 16,
  serverName: 'BDNODO01\\POINTDB',
  procName: '',
  precedingErrors: [
    RequestError: Invalid column name 'FechaCierre'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    },
    RequestError: Invalid column name 'Credito'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    }
  ]
}
Error al buscar usuario: QueryFailedError: Error: Invalid column name 'Consignacion'.
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\typeorm\driver\sqlserver\SqlServerQueryRunner.js:200:30
    at C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\base\request.js:440:25
    at Request.userCallback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:492:15)
    at Request.callback (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\request.js:239:14)
    at Parser.onEndOfMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:2653:22)
    at Object.onceWrapper (node:events:632:28)
    at Parser.emit (node:events:518:28)
    at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:25:12)
    at Readable.emit (node:events:518:28)
    at endReadableNT (node:internal/streams/readable:1698:12) {
  query: 'SELECT "Usuario"."idUsuario" AS "Usuario_idUsuario", "Usuario"."Nombre" AS "Usuario_Nombre", "Usuario"."Clave" AS "Usuario_Clave", "Usuario"."idGrupo" AS "Usuario_idGrupo", "Usuario"."Activo" AS "Usuario_Activo", "Usuario"."FechaCierre" AS "Usuario_FechaCierre", "Usuario"."Credito" AS "Usuario_Credito", "Usuario"."Consignacion" AS "Usuario_Consignacion" FROM "Usuario" "Usuario" WHERE ("Usuario"."Nombre" = @0 AND "Usuario"."Activo" = @1 AND "Usuario"."Clave" = @2) ORDER BY (SELECT NULL) OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY',
  parameters: [ 'KHERAS', 1, '1600567265' ],
  driverError: RequestError: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    code: 'EREQUEST',
    originalError: Error: Invalid column name 'Consignacion'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      info: [ErrorMessageToken]
    },
    number: 207,
    lineNumber: 1,
    state: 1,
    class: 16,
    serverName: 'BDNODO01\\POINTDB',
    procName: '',
    precedingErrors: [ [RequestError], [RequestError] ]
  },
  code: 'EREQUEST',
  originalError: Error: Invalid column name 'Consignacion'.
      at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 207,
      state: 1,
      class: 16,
      message: "Invalid column name 'Consignacion'.",
      serverName: 'BDNODO01\\POINTDB',
      procName: '',
      lineNumber: 1
    }
  },
  number: 207,
  lineNumber: 1,
  state: 1,
  class: 16,
  serverName: 'BDNODO01\\POINTDB',
  procName: '',
  precedingErrors: [
    RequestError: Invalid column name 'FechaCierre'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    },
    RequestError: Invalid column name 'Credito'.
        at handleError (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\mssql\lib\tedious\request.js:384:15)
        at Connection.emit (node:events:518:28)
        at Connection.emit (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\connection.js:970:18)
        at RequestTokenHandler.onErrorMessage (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\handler.js:284:21)
        at Readable.<anonymous> (C:\ProyectoNode\app\BACKPOINTAPP\BACKPOINTAPP\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
        at Readable.emit (node:events:518:28)
        at addChunk (node:internal/streams/readable:561:12)
        at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
        at Readable.push (node:internal/streams/readable:393:5)
        at nextAsync (node:internal/streams/from:194:22) {
      code: 'EREQUEST',
      originalError: [Error],
      number: 207,
      lineNumber: 1,
      state: 1,
      class: 16,
      serverName: 'BDNODO01\\POINTDB',
      procName: ''
    }
  ]
}
